starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/setup/ListItem.java

```diff
@@ -0,0 +1,22 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2.setup;
+
+/**
+ * Created by justinwells on 10/25/16.
+ */
+
+public class ListItem {
+    String text;
+
+    public String getText() {
+        return text;
+    }
+
+    public void setText(String text) {
+        this.text = text;
```

Comment:
Totally fine to have a setter here, but I just want to make sure you know you don't always need to have setters. Sometimes you don't want an object's values to change after it's created, so you might prefer not to provide any setters.